File: bentoml/docs/source/guides/snippets/grpc/go/request.go
package main

import (
	pb "github.com/bentoml/bentoml/grpc/v1"
)

var req = &pb.Request{
	ApiName: "classify",
	Content: &pb.Request_Ndarray{
		Ndarray: &pb.NDArray{
			Dtype:       *pb.NDArray_DTYPE_FLOAT.Enum(),
			Shape:       []int32{1, 4},
			FloatValues: []float32{3.5, 2.4, 7.8, 5.1},
		},
	},
}

File: docs/source/build-with-bentoml/snippets/grpc/python/request.py
from bentoml.grpc.v1 import service_pb2 as pb

req = pb.Request(
    api_name="classify",
    ndarray=pb.NDArray(
        dtype=pb.NDArray.DTYPE_FLOAT, shape=(1, 4), float_values=[5.9, 3, 5.1, 1.8]
    ),
)
